Syntaxes/Ruby.plist

```diff
@@ -926,37 +926,19 @@
 		</dict>
 		<dict>
 			<key>begin</key>
-			<string>(?x)
-			   (?:
-			     ^                      # beginning of line
-			   | (?&lt;=                   # or look-behind on:
-			       [=&gt;~(?:\[,|&amp;;]
-			     | [\s;]if\s			# keywords
-			     | [\s;]elsif\s
-			     | [\s;]while\s
-			     | [\s;]unless\s
-			     | [\s;]when\s
-			     | [\s;]assert_match\s
-			     | [\s;]or\s			# boolean opperators
-			     | [\s;]and\s
-			     | [\s;]not\s
-			     | [\s.]index\s			# methods
-			     | [\s.]scan\s
-			     | [\s.]sub\s
-			     | [\s.]sub!\s
-			     | [\s.]gsub\s
-			     | [\s.]gsub!\s
-			     | [\s.]match\s
-			     )
-			   | (?&lt;=                  # or a look-behind with line anchor:
-			        ^when\s            # duplication necessary due to limits of regex
-			      | ^if\s
-			      | ^elsif\s
-			      | ^while\s
-			      | ^unless\s
-			      )
-			   )
-			   \s*((/))(?![*+{}?])
+			<string>
+				(?x)
+				(?&lt;![\w)])((/))(?![?*+])
+				(?=
+					(?:\\/|[^/])*+          # Do NOT change the order
```

Comment:
looks like `/\\/` is not highlighted by this rule, but at least does not ruin the highlighting of the rest of the file, as in the issue's example. And I think that this can be fixed with
`(?:\\\\|\\/|[^/])*+`